tests/IntegrationTests/CampaignBuilderTest.php

```diff
@@ -0,0 +1,75 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\Matej\IntegrationTests;
+
+use Lmc\Matej\Exception\RequestException;
+use Lmc\Matej\Model\Command\Sorting;
+use Lmc\Matej\Model\Command\UserRecommendation;
+
+/**
+ * @covers \Lmc\Matej\RequestBuilder\CampaignRequestBuilder
+ */
+class CampaignBuilderTest extends IntegrationTestCase
```

Comment:
Maybe `@group integration` could be added. For fine grained test execution.

Reply:
I agree, but I also suggest keeping this as-is for now - before we do #43 I plan to setup phpunit.xml so that IntegrationTests directory will be in separate suite (`phpunit --suite integration`), so you don't need to define the test groups like this.